package.json

```diff
@@ -22,7 +22,11 @@
   "author": "Angular2-Meteor (https://github.com/Urigo/angular2-meteor)",
   "license": "MIT",
   "scripts": {
-    "build": "gulp build",
+    "typings": "typings i",
+    "prebuild": "npm run typings && npm run lint",
+    "build": "npm run build-only",
+    "build-only": "tsc",
```

Comment:
So can you change it `tsc || echo not ok`? don't know how come `tsc || 0` works for you